api-1.10/pom.xml

```diff
@@ -18,7 +18,7 @@
         <MODULE_NAME>${project.parent.name}</MODULE_NAME>
         <MODULE_VERSION>${project.parent.version}</MODULE_VERSION>
         <MODULE_PACKAGE>${project.parent.groupId}.${project.parent.artifactId}</MODULE_PACKAGE>
-        <openMRSBuildVersion>1.10.0</openMRSBuildVersion>
```

Comment:
Is there any reason why you switched from the 1.10.x series to the 1.11.x series?

Reply:
yes.You advised me to use 1.11x series so as to avail DbSession and DbSessionFactory.

Reply:
No i did not say that.

Reply:
okay.I tried 1.10.x series,DbSession and DbSessionFactory were not available so i had to use 1.11.x series

Reply:
Which exact version did you try?

Reply:
i resumed trying on 1.10.x series versions and got 1.10.4 having a success build,its what i have commited